tests/test_decorators.py

```diff
@@ -668,3 +668,77 @@ def make_item(foo, bar):
     assert '_schema' in errors
     assert len(errors['_schema']) == 1
     assert errors['_schema'][0] == 'preloadmsg1'
+    schema = ExampleSchema(strict=True)
```

Comment:
This is just one test nightmare. It would be better to split (at least yours) tests into individual tests making it obvious with test names what invariant you are testing. It will be easier to see guaranteed invariants and figure out what invariants still lack tests.